packages/did-auth-siop-adapter/lib/did/DidJWT.ts

```diff
@@ -238,7 +238,7 @@ function assertIssSelfIssuedOrDid(payload: JWTPayload) {
   }
 }
 
-export function getSubDidFromPayload(payload: JWTPayload, header?: JWTHeader): string {
+export function getSubDidFromPayload(payload: JWTPayload, header?: JWTHeader): string | undefined {
```

Comment:
Please revert. The assert method below ensures a string is always present

Reply:
I see it now. But my tsc does not look inside assertIssSelfIssuedOrDid, so have to return payload.sub! instead